src/custom-element.js

```diff
@@ -499,7 +515,9 @@ function createBaseCustomElementClass(win) {
       }, reason => {
         this.signals_.rejectSignal(CommonSignals.BUILT,
             /** @type {!Error} */ (reason));
-        reportError(reason, this);
+        if (!isBlockedByConsent(reason)) {
+          reportError(reason, this);
+        }
```

Comment:
Not crazy about checking `isBlockedByConsent` in three different places. I think we probably can introduce another class of errors, e.g. set `error.silent = true` like what we do for expected errors.

This is fine for now though.